tracks/javascript/exercises/leap/mentoring.md

```diff
@@ -47,7 +47,8 @@ keep improving:
 
     Hi there {name},
 
-    I have a few comments for you:
+    Congrats on submitting your first side exercise solution for the JavaScript
```

Comment:
Can you please move `leap` changes to separate PR?

Reply:
There are no leap changes, it's because of the incorrect rebase -- will be gone when merging.